sparse_ho/algo/forward.py

```diff
@@ -19,17 +23,48 @@ def __init__(self, use_stop_crit=True, verbose=False):
         self.use_stop_crit = use_stop_crit
         self.verbose = verbose
 
-    def get_beta_jac_v(
-            self, X, y, log_alpha, model, v, mask0=None, dense0=None,
+    def compute_beta_grad(
+            self, X, y, log_alpha, model, get_v, mask0=None, dense0=None,
             quantity_to_warm_start=None, max_iter=1000, tol=1e-3,
             full_jac_v=False):
-        mask, dense, jac = get_beta_jac_iterdiff(
+        """Compute beta and hypergradient, with forward differentiation of
+        proximal coordinate descent.
+
+        Parameters
+        ----------
+        X: array-like, shape (n_samples, n_features)
+            Design matrix.
+        y: ndarray, shape (n_samples,)
+            Observation vector.
+        log_alpha: float or np.array, shape (n_features,)
+            Logarithm of hyperparameter.
+        model:  instance of ``sparse_ho.base.BaseModel``
+            A model that follows the sparse_ho API.
+        get_v: callable
+            Function which return the values of the the vector v.
```

Comment:
1) typo: "the the"
2) can we detail a bit more what is  v?

Reply:
I changed for
`Function which return the values of the vector v.
 v is the gradient of the outer criterion.`

Reply:
What do you think of `Function which returns the gradient of the outer criterion` ?